lms/djangoapps/completion/models.py

```diff
@@ -25,12 +25,62 @@
 def validate_percent(value):
     """
     Verify that the passed value is between 0.0 and 1.0.
+
```

Comment:
nit: why add this empty line?

Reply:
To see if I can catch you off guard. Apparently I can not! I think at the moment I was making comments uniform throughout the code. I've removed it.